### PR TITLE
doc(cache): Documentation for cd caching agents config

### PR DESCRIPTION
**Pull request by german-muzquiz**
Resolves issue: [ENG-5775]

[ENG-5775]: https://armory.atlassian.net/browse/ENG-5775

### CODE REVIEW

File: content/en/docs/armory-admin/caching-agents.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Configuring clouddriver caching agents
```
Comment (ncknt) [inline content/en/docs/armory-admin/caching-agents.md]: Capitalize title

File: content/en/docs/armory-admin/caching-agents.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Depending on the number of elements of the infrastructure being cached, you would require to increase the number of replicas of clouddriver and/or increase its cpu and memory limits.
+
+The following configuration settings affect the behavior of the caching agents and can be used to adjust them depending on the size of the infrastructure being cached. If using the Spinnaker Operator, these settings live under the key `.spec.spinnakerConfig.profiles.clouddriver` of `SpinnakerService` manifest. If using Halyard, they live in `.hal/<profile name>/profiles/clouddriver-local.yml`.
+
+### SQL global caching agents configuration
+
+|Key|Description|Default Value|
+|----|---|---|
+|sql.scheduler.enabled|Enable or disable the sql scheduler|false
```
Comment (ncknt) [inline content/en/docs/armory-admin/caching-agents.md]: Wrap options in backticks? That's the style I used for agent options so it'd be nice to be consistent.

**Comment by ncknt** [comment]
I'd add a note that the `sql.connectionPools` needs to be configured with a link to https://docs.armory.io/docs/armory-admin/clouddriver-sql-configure/

Ideally the page would also include what metrics to check to know when to scale horizontally (or a link to a page that mentions it). It can obviously be a later improvements but if you already have such a page, link it!

You could also link to this blog post that goes over caching agents: https://www.armory.io/blog/deep-dive-into-clouddriver/ (the author is amazing by the way!)

**Comment by aimeeu** [comment]
Talked to Nico. Created a new page for caching agents concepts based on Nico's 2019 blog post.  Renamed files. Linked concepts and config.
Will do another pass on Monday, March 1.

**Pull request merged**